### ISSUE TITLE
codesign MacOS binaries

### ISSUE THREAD

**Issue by jonmmease** [plotly/Kaleido#1]
We should work out how to digitally sign the MacOS kaleido binary so that it doesn't get blocked by the default settings of Gatekeeper.

Looks like we could do this from the command using the `codesign` command after the build has been completed: https://developer.apple.com/library/archive/documentation/Security/Conceptual/CodeSigningGuide/Procedures/Procedures.html

**Comment by gvwilson** [comment]
closing as no longer relevant with TNG architecture
